tests/langtest_ir_lowering.rs

```diff
@@ -0,0 +1,80 @@
+//! A suite for testing lowerings of LLVM IR to Yk IR.
+
+use lang_tester::LangTester;
+use regex::Regex;
+use std::{
+    env,
+    fs::read_to_string,
+    path::{Path, PathBuf},
+    process::Command,
+};
+use tempfile::TempDir;
+use ykbuild::ykllvm_bin;
+
+const COMMENT: &str = ";";
+
+fn main() {
+    println!("Running IR lowering tests...");
+
+    let filter: fn(&Path) -> bool = |p| {
```

Comment:
Why break this out in an `fn` rather than a closure in `test_file_filter`? I think you can do something simpler like `test_file_filter(|p| p.extension() == Some("ll"))`?

Reply:
It's because I copied the code from another suite and didn't do the best job of minimising the code :)

Will fix.

Reply:
Maybe fix both at the same time, if they're in the yk repo?

Reply:
I think [aef79a4](https://github.com/ykjit/yk/pull/870/commits/aef79a41a54ebfb3bb1d87efaa8535c2c20f76c2) is right?

It's a bit miserable because `OsStr` doesn't coerce with `str` as I'd hoped.

Reply:
That looks much better to me! Is it the only instance of this in the repo?

Reply:
4610e93bd4b2 attempts to apply the same idioms, but it's not as clear cut for `langtest_c.rs`. Let me know what you think.